analytics.json

```diff
@@ -0,0 +1,7 @@
+[
+  "appnexus",
+  "ga",
+  "pulsepoint",
+  "roxot",
+  "sharethrough_analytics"
```

Comment:
Removed all the analytics adapter entries in the file. Though I think, it is probably a good idea to bundle all of them in the default build.